Makefile

```diff
@@ -29,11 +29,8 @@ package:
 deps:
 	go get github.com/mitchellh/gox
 	go get golang.org/x/tools/cmd/stringer
-	@go version | grep 1.4 ; if [ $$? -eq 0 ]; then \
-		echo "Installing godep and restoring dependencies"; \
-		go get github.com/tools/godep; \
-		godep restore; \
-	fi
+	go get -u github.com/kardianos/govendor
```

Comment:
This invokes a network call and git sync each time you run make. I recommend no `-u` or wrap in `if` block.